Summer Release EC-Earth Tasks

valentinaschueller opened the pull request:
contains 15 processing and one presentation task for monitoring EC-Earth 4 experiments.

uwefladrich commented:
Hi Valentina,
Thanks for the pull request!
I think that the Sphinx build directory is not needed under source control. Just the `Makefile` and `sources` is enough. Would you consider removing `build` (and `make.bat`) from the repository? Alternatively, if you want to keep them in your repo, I can remove the files from the pull request.

valentinaschueller commented:
Both are now gone from the repository!

uwefladrich commented:
> Both are now gone from the repository!

I still see `docs/sphinx/source/_build` in the Pull request and in your repository.

uwefladrich commented:
I think the actual monitoring code can be merged. If we want to make further changes, we can handle that with issues in your repository and new PR.
However, I would like to generalise the documentation. Nothing fundamental, just so that the docs cover all EC-Earth tasks, not just monitoring. Since the only non-monitoring task is `slurm/sbatch` at the moment, this means little changes. Just the title of the docs ("EC-Earth Monitoring" --> "ScriptEngine EC-Earth Tasks" or something similar), one more sentence or so on the index page (basically stating what's currently in the package, i.e. mostly monitoring and little else), and adding information about `slurm/sbatch` task. I can help with the actual changes.

valentinaschueller commented:
The `_build` folder is removed and I have updated the documentation. I added a file `docs/sphinx/source/tasks/sbatch.rst` where the documentation for the Sbatch/Slurm task can go. I added a short explanation to `index.rst`:
"This package contains ScriptEngine tasks for the EC-Earth 4 climate model.
Currently, it contains the EC-Earth 4 monitoring tool, as well as the Sbatch task for Slurm job creation."
